templates/terraform/examples/gkehub_membership_basic.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_container_cluster" "primary" {
+  name               = "<%= ctx[:vars]['cluster_name'] %>"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  provider = google-beta
+}
+
+resource "google_gke_hub_membership" "membership" {
+  membership_id = "<%= ctx[:vars]['name'] %>"
+  external_id = "aaaaaaaaa"
```

Comment:
Let's not set this. This will be updated server-side. We don't expect Membership clients for GKE clusters to know this at creation time (it will be updated to the kube-system namespace UID).